### DISCUSSION TITLE
Permissions

### DISCUSSION THREAD

**Discussion by ebardie** [Frederick888/external-editor-revived#57] [Ideas]
Thunderbird just asked for more permissions for EER 0.4.0.

I get a little curious when an extension asks for more permissions.

Can you explain why these are needed? Probably a good idea to include this in the release notes too.

**Comment by Frederick888** [comment]
Wiki...... Sigh

**Reply by ebardie** [reply]
Sorry for the extra work load caused by my missing the wiki changes. Great wiki BTW! 

I was trying to be polite and make a helpful suggestion (that some users at least look at the release notes for this kind of change info). I am sorry you feel I just wasted your time. "sigh" indeed.

Thank you for all the work you put in to this extension. It is really useful to me, and I appreciate that someone has borught this functionality back to Tbird.

Sorry again for missing the wiki.
